server/db/migrations/20220922101628_produce.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('produce', (table) => {
+    table.increments('id').primary()
+    table.string('name')
+    table.string('display_name')
+    table.string('type')
+    table.string('image')
```

Comment:
I would recommend changing this to `image_url` rather than `image` just to make it clear that you are returning a url not a image
```suggestion
    table.string('image_url')
```